moveos/moveos-stdlib/move-stdlib/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "MoveStdlib"
```

Comment:
If move-stdlib is directly moved to moveos-stdlib, if move-stdlib needs to be upgraded later, it can only be replaced by source code?

Reply:
And corresponding move rust code, or rely on cargo package management dependencies?

Reply:
move-stdlib in 0x1, moveos-stdlib in 0x2, there are two package. We need to find a way to support building remote package.